docs/settings/reporting-settings.asciidoc

```diff
@@ -203,26 +203,35 @@ The `file:` protocol is always denied, even if no network policy is configured.
 ==== CSV settings
 
 [[xpack-reporting-csv]] `xpack.reporting.csv.maxSizeBytes` {ess-icon}::
-The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `10mb`.
+The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `250mb`.
 
 [NOTE]
 ============
-Setting `xpack.reporting.csv.maxSizeBytes` much larger than the default 10 MB limit has the potential to negatively affect the
-performance of {kib} and your {es} cluster. There is no enforced maximum for this setting, but a reasonable maximum value depends
-on multiple factors:
+We recommend using CSV reports to export moderate amounts of data only. The feature enables analysis of data in external tools, but it's not intended for bulk export or to backup {es} data. Setting `xpack.reporting.csv.maxSizeBytes` to larger than 250 MB can negatively affect the performance of your {es} cluster.
 
-* The `http.max_content_length` setting in {es}.
-* Network proxies, which are often configured by default to block large requests with a 413 error.
-* The amount of memory available to the {kib} server, which limits the size of CSV data that must be held temporarily.
+The following deployment configurations may lead to failed report jobs or incomplete reports:
 
-For information about {kib} memory limits, see <<production, using {kib} in a production environment>>.
+* Any shard needed for search is unavailable.
+* Data is stored on slow storage tiers. 
+* Network latency between nodes is high.
+* {ccs-cap} is used.
+
+To export large amounts of data we recommend using {es} APIs directly. See {ref}/point-in-time-api.html[Point in time API], or {ref}/sql-rest-format.html#_csv[SQL with CSV response data format].
 ============
 
 `xpack.reporting.csv.scroll.size`::
 Number of documents retrieved from {es} for each scroll iteration during a CSV export. Defaults to `500`.
+[NOTE]
+============
+If the default size of 500 covers a large number of shards in {es}, you may need to lower this setting. 
+============
 
 `xpack.reporting.csv.scroll.duration`::
  Amount of {time-units}[time] allowed before {kib} cleans the scroll context during a CSV export. Defaults to `30s`.
+[NOTE]
+============
+If search latency in {es} is sufficiently high, such as if you are using {ccs}, you may need to increase the setting.
```

Comment:
One advantage of using PIT is that you can refresh the PIT independently of the searches. `30s` seems generous for all non-broken situations anyway, even if using CCS, but why not set it higher by default rather than asking users to think about the search latency themselves?

Reply:
40 minutes between searches?! Wow. Was that because ES was taking tens-of-minutes to respond to each search? If so that deserves further investigation for sure. In any case I would expect it to be possible to keep the PIT alive via other requests in the meantime, rather than asking the user to judge what "sufficiently high" might mean in this context.